### PR TITLE
add DAOSquare space

### PR DESCRIPTION
**Pull request by taosue**
Add DAOSquare space

### CODE REVIEW

**Comment by bonustrack** [comment]
@taosue the logo is wrong

**Comment by taosue** [comment]
> @taosue the logo is wrong

hey , I have correct it

File: spaces/daosquare/index.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "key": "daosquare",
+  "symbol": "RICE",
+  "name": "DAOSquare",
+  "defaultView": "core",
+  "address": "0xEc0B6AFb3F9a609CEEd67e2CA551A4c573fd45F7",
+  "token": "0xEc0B6AFb3F9a609CEEd67e2CA551A4c573fd45F7",
+  "core": [],
```
Comment (bonustrack) [inline spaces/daosquare/index.json:8]: I recommend you to add some addresses here otherwise change default view to "all"

Comment (taosue) [reply spaces/daosquare/index.json:8]: hey , I have slove it and review it plz

**Review by bonustrack — APPROVED** [review]

File: spaces/daosquare/index.json (hunk 1 above)
Comment (taosue) [reply spaces/daosquare/index.json:8]: hey , I have slove it and review it plz @bonustrack

**Pull request merged**